src/services/userService.js

```diff
@@ -1,13 +1,12 @@
+import { user } from './mocks/userData';
+
 export function logIn({login, password}) {
 	return new Promise((resolve, reject) => {
-		setTimeout(() => {
-			if(login && password) {
-				resolve("You are successfully logged in")
-			} else {
-				reject("Fields are empty")
-			}
-		}, 1000);
-				
+		if(user && login && password && login === user.login && password === user.password) {
```

Comment:
This check should be called in form before you call logIn method.
You shouldn't call this method without login or password, because it doesn't have any sense.
In this method you should only check if password and login matched.